lib/cisco_node_utils/router_ospf_area.rb

```diff
@@ -185,6 +187,149 @@ def default_filter_list_out
       config_get_default('ospf_area', 'filter_list_out')
     end
 
+    # CLI can be the following or none
+    # area 1.1.1.1 nssa
+    # the above command can be appended with no-summary and/or
+    # no-redistribution and/or default-information-originate.
+    # route-map <map> can be appended with default-information-originate
+    # Basically, every property this CLI configures is optional
+    def nssa
+      hash = {}
+      output = config_get('ospf_area', 'nssa', @get_args)
+      return hash if output.nil?
+      output.each do |line|
+        next if line.include?('translate')
+        hash[:nssa_enable] = true
```

Comment:
I don't quite understand why you need this `nssa_enable`. Is this an explicit property that will show up in the manifest?